contrib_bots/contrib_server.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import absolute_import
+from flask import Flask, request, jsonify
+from run import get_lib_module
+from bot_lib import BotHandlerApi, StateHandler
+import os
+import sys
+import json
+from typing import Any, Dict, Mapping, Union
+from werkzeug.exceptions import BadRequest
+from six.moves.configparser import SafeConfigParser
+
+our_dir = os.path.dirname(os.path.abspath(__file__))
+
+# For dev setups, we can find the API in the repo itself.
+if os.path.exists(os.path.join(our_dir, '../api/zulip')):
+    sys.path.insert(0, '../api')
+
+from zulip import Client
+
+config_file_path = sys.argv[1]
```

Comment:
Yeah it would be much better to shift these code lines into separate functions and call them from `if __name__ == "__main__"` block